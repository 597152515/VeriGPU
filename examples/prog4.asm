out abx
out cdx
out 12x
out 34x
li x1 01x
li x1 02x
li x1 dex
li x1 adx
halt

20x:
word 0000deadx
word 0000beefx
