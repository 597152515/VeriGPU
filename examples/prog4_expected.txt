out 0 00000000000000000000000000011011 0000001b 27
out 1 00000000000000000000000000101101 0000002d 45
out 2 00000000000000000000000000110010 00000032 50
out 3 00000000000000000000000001000100 00000044 68